vmdb/app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -2,6 +2,8 @@
 
 module EmsRefresh::Parsers
   class Openstack < Cloud
+    include EmsRefresh::Parsers::OpenstackCommon::OrchestrationStacks
```

Comment:
I made a comment in https://github.com/ManageIQ/manageiq/pull/1931 about changing the namespace here to `EmsRefresh::Parsers::Openstack` instead of `EmsRefresh::Parsers::OpenstackCommon` and we'll just reuse the `Openstack` name for the shared inventory parsing code.